Styles/components.css

```diff
@@ -366,52 +393,63 @@ body {
 }
 
 /* ********************input********************************* */
+.input-group {
+  display: flex;
+  flex-direction: column;
+}
+.input-label {
+  margin-bottom: 0.3em;
+}
 .input {
   font-size: 1rem;
   padding: 0.3rem;
   background-color: transparent;
-  color: white;
+  color: rgba(0, 0, 0, 0.911);
   font-family: inherit;
   outline: none;
   box-shadow: none;
-  border-style: ridge;
+  border: 1px ridge #4fe7ad;
   border-radius: 0.25rem;
 }
+.input-sm {
+  width: 25%;
+}
 .input-md {
   width: 50%;
 }
 .input-lg {
   width: 90%;
 }
+.input:focus {
+  border-bottom: 2px ridge var(--myColor);
+}
+
 .line-input {
   border-top: none;
   border-right: none;
   border-left: none;
-  border-bottom: 2px ridge #4d6a7c81;
+  border-bottom: 2px ridge #11f79f86;
```

Comment:
why this color uses hardcoded color,just check if it is in the variables already use else if this color is not repeating ,I am ok with this